nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Utilities/JsonHelper.cs

```diff
@@ -149,9 +149,10 @@ public static string UpdateJsonProperty(string json, string[] propertyPath, stri
             var updatedJsonLines = resultJson.Split('\n').Select(l => l.TrimEnd('\r')).ToArray();
             for (int i = 0; i < Math.Min(originalJsonLines.Length, updatedJsonLines.Length); i++)
             {
-                if (updatedJsonLines[i].EndsWith(",") && !originalJsonLines[i].EndsWith(","))
+                var updatedLine = updatedJsonLines[i];
+                if (updatedLine.EndsWith(',') && updatedLine.Contains("//", StringComparison.Ordinal) && !originalJsonLines[i].EndsWith(','))
```

Comment:
I can't see an obvious way to do this while using `Utf8JsonReader` - it seems to skip over them and never produce a token that represents a completely blank line. I think trying to fix that too would require more workarounds, and I'd rather just fix the scenario and stop it producing invalid JSON, rather than preserve the blank line formatting too.

Reply:
Sorry, I am not saying we should preserve blank lines. I am suggesting remove blank lines from our original json, since we know they will be removed from the updated json. Then this comparison will line up better.

```
var originalJsonLines = json.Split('\n').Select(l => l.TrimEnd('\r')).Where(l => !l.IsNullOrEmpty()).ToArray();
```